peer-mgmt-service/cmd/pms/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"golang.org/x/exp/slog"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/infrastructure-services/peer-mgmt-service/pkg/config"
+	"github.com/ethereum-optimism/infrastructure-services/peer-mgmt-service/pkg/service"
```

Comment:
Seems like all imports should point to this local, public repo instead of pulling from the private one